src/test/groovy/io/openliberty/tools/gradle/TestMultiModuleLooseEarAppDevMode.groovy

```diff
@@ -0,0 +1,65 @@
+/*
+ * (C) Copyright IBM Corporation 2020, 2023.
```

Comment:
change copyright to just say 2024 or remove it entirely (I personally don't think test cases need a copyright).

Reply:
Removed Copyright